service/src/main/java/cz/fi/muni/pa165/service/RoomServiceImpl.java

```diff
@@ -0,0 +1,80 @@
+package cz.fi.muni.pa165.service;
+
+import cz.fi.muni.pa165.dao.HotelDao;
+import cz.fi.muni.pa165.dao.RoomDao;
+import cz.fi.muni.pa165.entity.Hotel;
+import cz.fi.muni.pa165.entity.Room;
+import cz.fi.muni.pa165.enums.RoomType;
+
+import java.util.List;
+import javax.inject.Named;
+import javax.inject.Inject;
+import java.util.stream.Collectors;
+
+/**
+ * @author Peter Neupauer
+ */
+@Named
+public class RoomServiceImpl implements RoomService {
+
+    @Inject
+    private RoomDao roomDao;
+
+    @Inject
+    private HotelDao hotelDao;
+
+    @Override
+    public Room findById(Long id) {
+        return roomDao.findById(id);
+    }
+
+    @Override
+    public List<Room> findByType(RoomType type) {
+        return roomDao
```

Comment:
Tohle fitrovani by se melo delat v DB takze v DAO. Takhle se ti do pameti natahnou zbytecne vsechny Room. Stejne i pro `findByHotel(Hotel hotel)`

Reply:
ja vime :) ale nechcelo sa mi to prerabat